openapi/components/schemas/GroupPost.yaml

```diff
@@ -3,28 +3,21 @@ type: object
 properties:
   id:
     $ref: ./NotificationID.yaml
-    nullable: true
   groupId:
     $ref: ./GroupID.yaml
-    nullable: true
   authorId:
     $ref: ./UserID.yaml
-    nullable: true
   editorId:
     $ref: ./UserID.yaml
     nullable: true
   visibility:
     $ref: ./GroupPostVisibility.yaml
-    nullable: false
   roleId:
     $ref: ./GroupRoleIDList.yaml
-    nullable: false
   title:
     type: string
-    nullable: true
   text:
```

Comment:
Did you intend to remove this, and others like it that you marked as nullable?

Reply:
Yes, I don't think they're nullable by what the front end does. I did the least amount of options on a post and only 3 fields remained Null. Everything else was required or auto-filled by the js.

![image](https://github.com/vrchatapi/specification/assets/44061648/6dbddefb-8d2c-4b43-ada0-74c3ec24903d)